src/schema/objects/datatypes.yaml

```
@@ -34,6 +35,11 @@ func:
   display_name: Task-Based Magnetic Resonance Imaging
   description: |
     Task (including resting state) imaging data
+icephys:
+  value: icephys
+  display_name: Intracellular Electrophysiology
+  description: |
+    Electrophysiological data acquired from an individual cell
```

```suggestion
    Electrophysiological data acquired from individual cells
```

---

This may not be completely unambiguous, under certain circumstances it could be taken to apply also to extracellular recordings. How about
"Electrophysiological data acquired from intracellular or patch microelectrodes"
This would parallel the description of ecephys,

---

good idea, doing suggestion on your behalf with your wording:
```suggestion
    Electrophysiological data acquired from intracellular or patch microelectrodes
```